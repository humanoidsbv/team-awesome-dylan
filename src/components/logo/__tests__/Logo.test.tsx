import * as React from 'react';
import { shallow } from 'enzyme';

import Logo from '../Logo';

it('matches snapshot', () => {
  const logo = shallow(
    <Logo />
  );
  expect(logo).toMatchSnapshot();
});
