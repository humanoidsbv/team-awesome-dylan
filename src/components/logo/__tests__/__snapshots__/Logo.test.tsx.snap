// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<a
  className="logo"
  href="/"
>
  team awesome
</a>
`;
